hosted-cluster-config-operator/controllers/clusteroperator/setup.go

```diff
@@ -1,22 +1,37 @@
 package clusteroperator
 
 import (
+	"context"
+
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	configinformers "github.com/openshift/client-go/config/informers/externalversions"
+	"github.com/openshift/hypershift/hosted-cluster-config-operator/controllers"
 	"github.com/openshift/hypershift/hosted-cluster-config-operator/operator"
 )
 
 func Setup(cfg *operator.HostedClusterConfigOperatorConfig) error {
-	clusterOperators := cfg.TargetConfigInformers().Config().V1().ClusterOperators()
+	configClient, err := configclient.NewForConfig(cfg.TargetConfig)
+	if err != nil {
+		return err
+	}
+	informerFactory := configinformers.NewSharedInformerFactory(configClient, controllers.DefaultResync)
```

Comment:
Doesn't this duplicate the informer from the `clusterversion` controller below?

Reply:
Yes it does, but afaik we want to get rid of these individual controllers, no?